src/tests/SimTest.cpp

```diff
@@ -833,26 +835,34 @@ void SimTest::createMagnumRenderingOff() {
     // cast a ray at the object to check that the object is actually there
     auto raycastresults = simulator->castRay(
         esp::geo::Ray({10.0, 9.0, 10.0}, {0.0, 1.0, 0.0}), 100.0, 0);
-    CORRADE_VERIFY(raycastresults.hits[0].objectId == objectID);
+    CORRADE_COMPARE(raycastresults.hits[0].objectId, objectID);
     auto point = raycastresults.hits[0].point;
-    CORRADE_VERIFY(distanceBetween(point, {10.0, 9.9, 10.0}) < 0.001);
+    CORRADE_COMPARE_AS(distanceBetween(point, {10.0, 9.9, 10.0}), 0.001,
+                       Cr::TestSuite::Compare::Less);
     raycastresults = simulator->castRay(
         esp::geo::Ray({10.0, 11.0, 10.0}, {0.0, -1.0, 0.0}), 100.0, 0);
-    CORRADE_VERIFY(raycastresults.hits[0].objectId == objectID);
+    CORRADE_COMPARE(raycastresults.hits[0].objectId, objectID);
     point = raycastresults.hits[0].point;
-    CORRADE_VERIFY(distanceBetween(point, {10.0, 10.1, 10.0}) < 0.001);
+    CORRADE_COMPARE_AS(distanceBetween(point, {10.0, 10.1, 10.0}), 0.001,
+                       Cr::TestSuite::Compare::Less);
   };
 
   auto testBoundingBox = [&]() {
     // check that we can still compute bounding box of the object
     Magnum::Range3D meshbb = objectNode->getCumulativeBB();
     float eps = 0.001;
-    CORRADE_VERIFY(abs(meshbb.left() - -0.1) < eps);
-    CORRADE_VERIFY(abs(meshbb.right() - 0.1) < eps);
-    CORRADE_VERIFY(abs(meshbb.bottom() - -0.1) < eps);
-    CORRADE_VERIFY(abs(meshbb.top() - 0.1) < eps);
-    CORRADE_VERIFY(abs(meshbb.back() - -0.1) < eps);
-    CORRADE_VERIFY(abs(meshbb.front() - 0.1) < eps);
+    CORRADE_COMPARE_AS(abs(meshbb.left() - -0.1), eps,
+                       Cr::TestSuite::Compare::Less);
```

Comment:
Could be even

```suggestion
    CORRADE_COMPARE_AS(meshbb.left(), -0.1,
                       Cr::TestSuite::Compare::around(eps));
```

and similar below ... although i'm afraid i'm overdoing it a bit already :)

Reply:
around will still pass if the value is a bit higher than eps though, no?

Reply:
It will only pass if `abs(meshbb.left() - -0.1) < eps`, so basically the same as the original code but with smaller chance for an error in the comparison expression and with clearer failure diagnostic.